akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala

```diff
@@ -114,9 +114,8 @@ object OverflowStrategy {
 
   /**
    * If the buffer is full when a new element arrives, drops the new element.
-   *
-   * Deprecated since 2.6.11. Use Source.queue instead.
    */
+  @deprecated("Use Source.queue instead", "2.6.11")
```

Comment:
I think this API is used both from Scala and from Java. Since the `@deprecated` annotation is Scala-only, I think it might be good to annotate this method *both* with this `@deprecated` annotation *and* the Java `@Deprecated`. Perhaps its might even be good to also add the `@deprecated` javadoc tag to make sure the pointer to `Source.queue` makes it to the Javadocs.

Reply:
Thank you @raboof . I'll do it today.

Reply:
I have added annotation `Deprecated` and deprecated javadoc tag. The link renders correctly 

<img width="1107" alt="Снимок экрана 2021-12-16 в 20 43 18" src="https://user-images.githubusercontent.com/4740207/146422162-e46078de-bf4a-4633-aeb8-8e501688841d.png">